config/toml.go

```diff
@@ -349,8 +349,24 @@ dial_timeout = "{{ .P2P.DialTimeout }}"
 #   2) "v1" - prioritized mempool.
 version = "{{ .Mempool.Version }}"
 
+// Recheck (default: true) defines whether Tendermint should recheck the
```

Comment:
Thank you for adding the comments, they will be very helpful for anyone using Tendermint! 
For comments that go into the `.toml` file you have to start the line with `#` instead of `/`. I suspect the checks fail due to this. Can you please update this and rerun the CI checks?

Reply:
Thanks for the tip! Apologies for not noticing this error. Will fix shortly

Reply:
Resolved by https://github.com/tendermint/tendermint/pull/9598/commits/6141d2498298e48e5eb52fc9b2eb18ebfc3f89ec